lib/python/qmk/path.py

```diff
@@ -6,6 +6,7 @@
 
 from qmk.errors import NoSuchKeyboardError
 
+from bs4 import UnicodeDammit
```

Comment:
I'd say just remove the changes to this file. We don't want to pull in bs4 for this.